docs/source/duckdb_cli.md

```diff
@@ -0,0 +1,52 @@
+# DuckDB CLI
+
+The [DuckDB CLI](https://duckdb.org/docs/api/cli/overview.html) (Command Line Interface) is a single, dependency-free executable. 
+
+<Tip>
+
+For installation details, visit the [installation page](https://duckdb.org/docs/installation).
+
+</Tip>
+
+Starting from version `v0.10.3-dev1012`, the DuckDB CLI includes native support for accessing datasets on Hugging Face via URLs. Here are some features you can leverage with this powerful tool:
+
+- Query public, gated and private datasets
+- Analyze datasets and perform SQL operations
+- Process and transform datasets
+- Conduct vector similarity search on embedding datasets
+- Export datasets to other formats
+- Implement full-text search on datasets
+- And more! For a complete list of DuckDB features, visit the DuckDB documentation.
+
+To start the CLI, execute the following command in the installation folder:
+
+```bash
+./duckdb
+```
+
+## Forming the Hugging Face URL
+
+To access Hugging Face datasets, use the following URL format:
+
+```plaintext
+hf://datasets/{my-username}/{my-dataset}@~parquet/{path_to_parquet_file} 
```

Comment:
I would start without mentioning the refs/convert/parquet branch because it's very specific (only useful for datasets not in parquet or CSV)

Reply:
Maybe we can keep that for more advanced use cases (so later in the docs or in another page) ?

Reply:
Yes, better idea, I left it as hf://datasets/{my-username}/{my-dataset}/{path_to_parquet_file}  (as in DuckDB doc) and added a tip for it later.